### DISCUSSION TITLE
Set pen layer

### DISCUSSION THREAD

**Discussion by ThisHasFourWords** [ScratchAddons/ScratchAddons#7422] [Ideas]
Currently, the pen can only draw on the stage and can be obstructed by any sprite. Is there a way to add an extension that allows you to set the layer of pen (just like there's a block to set sprite layers) or have the pen layer match the respective sprite using the pen down block?

**Comment by BroJac5246** [comment]
>  Is there a way to add an extension that allows you to set the layer of pen

Unfortunately, no. Scratch doesn't support this and we wouldn't be able to add support for people who don't use our extension.
